src/Microsoft.Data/PrimitiveColumnArithmetic.cs

```diff
@@ -4,155 +4,157 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Generated from PrimitiveColumnArithmetic.tt. Do not modify directly
```

Comment:
Some of the VS generators have a banner like
```
//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a tool.
//     Runtime Version:2.0.50727.42
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
//------------------------------------------------------------------------------
```
I vaguely remember that the `<autogenerated>` part has some effect - on VS maybe - to prevent you writing to it? If so it is nice to put it in this form. I don't know though.

Reply:
I don't think this matters and you can ignore.